src/ladds/io/VTUWriter.cpp

```diff
@@ -168,8 +168,8 @@ std::string VTUWriter::filenamePayload(ConfigReader &config,
   autopas::AutoPas_MPI_Comm_size(decomposition.getCommunicator(), &numRanks);
   const auto maxDigitsIterations = std::to_string(config.getLastIterationNr()).size();
   std::stringstream ss;
-  ss << "Output_" << std::setfill('0') << std::setw(static_cast<int>(maxDigitsIterations)) << iteration << "_"
-     << std::setw(static_cast<int>(std::to_string(numRanks).size())) << rank << ".vtu";
+  ss << "Output_" << std::setw(static_cast<int>(std::to_string(numRanks).size())) << rank << "_" << std::setfill('0')
+     << std::setw(static_cast<int>(maxDigitsIterations)) << iteration << ".vtu";
```

Comment:
```suggestion
  ss << "Output_" << std::setfill('0') << std::setw(static_cast<int>(std::to_string(numRanks).size())) << rank << "_" 
     << std::setw(static_cast<int>(maxDigitsIterations)) << iteration << ".vtu";
```
We also want `0` as the fill char for the first fill. Not sure if `setfill` now only applies for the next fill operation or globally but I think the latter.